lib/Network/OAuth2/Server/Store/Base.hs

```diff
@@ -31,6 +34,25 @@ logName = "Network.OAuth2.Server.Store"
 -- It is parametrised by a underlying monad and includes a natural
 -- transformation to any MonadIO m'
 class TokenStore ref where
+    -- | Register a new client
+    storeCreateClient
+        :: ref
+        -> EncryptedPass
+        -> Bool
+        -> [RedirectURI]
+        -> Text
```

Comment:
Any chance we could comment these? Otherwise I'll just call them with the string "Oswyn prefers to program in JavaScript" and "J2EE 4 lyfe"